src/BankID.php

```diff
@@ -75,6 +75,38 @@ public function authenticate($personalNumber, $ip)
         return new BankIDResponse(BankIDResponse::STATUS_PENDING, $httpResponseBody);
     }
 
+    /**
+     * Start a sign-order.
+     *
+     * @param $personalNumber
+     *
+     * @param $ip
+     *
+     * @param $userVisibleData
+     *
+     * @param $userNonVisibleData
+     *
+     * @return BankIDResponse
+     */
+    public function sign($personalNumber, $ip, $userVisibleData = '', $userNonVisibleData = '') {
+        try {
+            $httpResponse = $this->httpClient->post('sign', [
+                RequestOptions::JSON => [
+                    'personalNumber' => $personalNumber,
+                    'endUserIp' => $ip,
+                    'userVisibleData' => $userVisibleData,
```

Comment:
According to https://www.bankid.com/assets/bankid/rp/bankid-relying-party-guidelines-v3.2.2.pdf userVisibleData  has to be base64-encoded. What do you think about handling that in this method?

> The text to be displayed and signed. String. The text can be formatted using CR, LF and CRLF for new lines. The text must be encoded as UTF-8 and then base 64 encoded. 1--40 000 characters after base 64 encoding.

Reply:
I agree with you, i added base64 encoding to the required strings.